<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-03-31T00:41:09 IST" name="MyStore Automation Suite" finished-at="2023-03-31T00:41:30 IST" duration-ms="20325">
    <groups>
    </groups>
    <test started-at="2023-03-31T00:41:09 IST" name="MyStore Test Cases" finished-at="2023-03-31T00:41:30 IST" duration-ms="20325">
      <class name="com.mystore.testcases.Search">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.mystore.testcases.Search@3febb011]" started-at="2023-03-31T00:41:10 IST" name="setup" finished-at="2023-03-31T00:41:20 IST" duration-ms="10274" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifySearchWithValidProduct()[pri:1, instance:com.mystore.testcases.Search@3febb011]" started-at="2023-03-31T00:41:20 IST" name="verifySearchWithValidProduct" finished-at="2023-03-31T00:41:22 IST" duration-ms="1493" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySearchWithValidProduct -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.mystore.testcases.Search@3febb011]" started-at="2023-03-31T00:41:22 IST" name="tearDown" finished-at="2023-03-31T00:41:22 IST" duration-ms="302" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.mystore.testcases.Search@3febb011]" started-at="2023-03-31T00:41:22 IST" name="setup" finished-at="2023-03-31T00:41:28 IST" duration-ms="6229" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifySearchWithInValidProduct()[pri:2, instance:com.mystore.testcases.Search@3febb011]" started-at="2023-03-31T00:41:28 IST" name="verifySearchWithInValidProduct" finished-at="2023-03-31T00:41:29 IST" duration-ms="1020" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[No Product message in search results is displayed expected [There is a product that matches the search criteria.] but found [There is no product that matches the search criteria.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: No Product message in search results is displayed expected [There is a product that matches the search criteria.] but found [There is no product that matches the search criteria.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at com.mystore.testcases.Search.verifySearchWithInValidProduct(Search.java:58)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySearchWithInValidProduct -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.mystore.testcases.Search@3febb011]" started-at="2023-03-31T00:41:29 IST" name="tearDown" finished-at="2023-03-31T00:41:30 IST" duration-ms="237" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="verifySearchWithoutAnyProduct()[pri:3, instance:com.mystore.testcases.Search@3febb011]" started-at="2023-03-31T00:41:30 IST" name="verifySearchWithoutAnyProduct" finished-at="2023-03-31T00:41:30 IST" depends-on-methods="com.mystore.testcases.Search.verifySearchWithInValidProduct" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.mystore.testcases.Search.verifySearchWithoutAnyProduct() on instance com.mystore.testcases.Search@3febb011 depends on not successfully finished methods 
[com.mystore.testcases.Search.verifySearchWithInValidProduct() on instance com.mystore.testcases.Search@3febb011]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.mystore.testcases.Search.verifySearchWithoutAnyProduct() on instance com.mystore.testcases.Search@3febb011 depends on not successfully finished methods 
[com.mystore.testcases.Search.verifySearchWithInValidProduct() on instance com.mystore.testcases.Search@3febb011]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySearchWithoutAnyProduct -->
      </class> <!-- com.mystore.testcases.Search -->
    </test> <!-- MyStore Test Cases -->
  </suite> <!-- MyStore Automation Suite -->
</testng-results>
